tests/functional/z_cluster/cluster_expansion/test_add_capacity.py

```
@@ -62,13 +62,27 @@ def add_capacity_test(ui_flag=False):
+    logger.info(f"Existing OSD pod count: {len(existing_osd_pod_names)}")
+
+    logger.test_step(
+        "Add capacity via UI"
+        if (ui_add_capacity_conditions() and ui_flag)
+        else "Add capacity via CLI"
+    )
     if ui_add_capacity_conditions() and ui_flag:
         result = ui_add_capacity(osd_size)
     else:
         result = storage_cluster.add_capacity(osd_size)
+    logger.info(f"Add capacity result (expected OSD replica sets): {result}")
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)